src/modules/sensors/camera.c

```diff
@@ -48,6 +55,7 @@ struct state {
     uint32_t pixelformat;
     double *brightness;
     struct buffer buf;
+    struct histogram hist;
```

Comment:
Why not using 
`struct histogram hist[HISTOGRAM_STEPS]`
instead?
Thus struct histogram would become just:
```
struct histogram {
    double count;
    double sum;
};
```